src/controllers/light-the-bridge.js

```diff
@@ -25,11 +29,15 @@ module.exports = async function(req, res) {
     ...tweet,
     ...{
       created_at: new Date().toISOString(),
-      color_description: '',
-      color1: '',
-      color2: '',
-      color3: '',
-      color4: '',
+      color_description: colors
+        .map(function(element) {
+          return _.capitalize(element.color)
```

Comment:
It's probably okay to return lower case color names.

Reply:
I left it as capitalized, but if you prefer it all lowercase I can change it too